Library/Formula/git-subrepo.rb

```diff
@@ -0,0 +1,40 @@
+class GitSubrepo < Formula
+  homepage "https://github.com/ingydotnet/git-subrepo"
+  head "https://github.com/ingydotnet/git-subrepo.git"
+
+  stable do
+    url "https://github.com/ingydotnet/git-subrepo/archive/0.2.1.tar.gz"
+    sha256 "2caa7f7c50af637b2338f0a6051401e6567ee059dc036eff7d859450a56865f1"
+
+    # Allow symbolic linking of git-subrepo script.
```

Comment:
A comment here as to when this can be removed may be useful. Upstream PR or simple "Remove on next stable release" or something.